### ISSUE TITLE
ubuntu/vivid64 box isn't available on Atlas

### ISSUE THREAD

**Issue by heartsucker** [freedomofpress/ansible-role-grsecurity#53]
[Link.](https://atlas.hashicorp.com/boxes/search?utf8=%E2%9C%93&sort=&provider=&q=ubuntu%2Fvivid64)

```
heartsucker@archimedes:~/code/freedom-of-the-press/ansible-role-grsecurity (master)$ vagrant up grsec-build
Bringing machine 'grsec-build' up with 'virtualbox' provider...
==> grsec-build: Box 'ubuntu/vivid64' could not be found. Attempting to find and install...
    grsec-build: Box Provider: virtualbox
    grsec-build: Box Version: >= 0
The box 'ubuntu/vivid64' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/ubuntu/vivid64"]
Error: The requested URL returned error: 404 Not Found
```

**Comment by garrettr** [comment]
This Canonical Launchpad [thread](https://bugs.launchpad.net/ubuntu/+bug/1553119) is not definitive, but hypothesizes that they're no longer being provided because Vivid has been end-of-lifed. Since Vivid has been end-of-lifed, we shouldn't be using it anyway as it is no longer receiving updates (including for security issues).

@conorsch We should switch to the next available LTS version of Ubuntu (as long as it has the kernel-package issue fixed) or switch to Debian for the build box. Is there any compelling reason to continue using Ubuntu over Debian?

**Comment by conorsch** [comment]
> Is there any compelling reason to continue using Ubuntu over Debian?

Unfortunately, yes—see #30 and #32. In sum, the Debian Vagrant images don't have sufficient disk space to compile a kernel, and attaching extra storage to the VirtualBox VM is a painful process that's difficult to automate reliably. 

Long-term solution should be to switch to the new Ubuntu LTS, which won't land for a few more weeks, so we're blocked there.

Short-term solution is to update the box URLs to use Canonical's directly, rather than proxying through the unreliable Hashicorp Atlas service (see #37). See #54 for an implementation of this. Testing it now, should have confirmation of the fix in a few hours (since builds take a while).

**Comment by conorsch** [comment]
Fixed in master, thanks to #54:

```
$ vagrant box remove --force --all ubuntu/vivid64
The box you requested to be removed could not be found. No
boxes named 'ubuntu/vivid64' could be found.
$ vagrant up grsec-build
Bringing machine 'grsec-build' up with 'virtualbox' provider...
==> grsec-build: Box 'ubuntu/vivid64' could not be found. Attempting to find and install...
    grsec-build: Box Provider: virtualbox
    grsec-build: Box Version: >= 0
==> grsec-build: Box file was not detected as metadata. Adding it directly...
==> grsec-build: Adding box 'ubuntu/vivid64' (v0) for provider: virtualbox
    grsec-build: Downloading: https://cloud-images.ubuntu.com/vagrant/vivid/current/vivid-server-cloudimg-amd64-vagrant-disk1.box
    grsec-build: Progress: 5% (Rate: 2754k/s, Estimated time remaining: 0:03:49)
```

Note that #55 is still an issue for the vivid64 box; in the meantime, just manually install `libssl-dev` and you're set.

**Comment by conorsch** [comment]
Resolved by #54, and the related issue #55 is now resolved, as well. Closing.
